Sage/carbon
# fix(date): datepicker weekdays not translated (FE-3027)

### Proposed behaviour
Translate Weeksdays in the Daypicker
![Screenshot 2020-10-22 at 15 21 43](https://user-images.githubusercontent.com/22885392/96877739-52a45880-147a-11eb-8179-01e2c47a798d.png)

### Current behaviour
Weekdays are not being translated in the DayPicker
![Screenshot 2020-10-22 at 15 14 11](https://user-images.githubusercontent.com/22885392/96876895-47046200-1479-11eb-8a9e-be892c2f098d.png)

### Checklist


~~- [ ] All themes are supported if required~~
~~- [ ] Cypress automation tests added or updated if required~~
~~- [ ] Storybook added or updated if required~~
~~- [ ] Typescript `d.ts` file added or updated if required~~
~~- [ ] Carbon implementation and Design System documentation are congruent~~

### Additional context
fixes #3120

### Testing instructions
1. open https://codesandbox.io/s/carbon-formik-yup-zoj6u?file=/src/index.js
2. check the weekdays
